content/docs/0.8.x/operate/install/index.md

```diff
@@ -222,25 +222,25 @@ Use this page if you have already Keptn experience and would like to install it
 
 Every Keptn release provides binaries for the Keptn CLI. These binaries are available for Linux, macOS, and Windows.
 
-- Download the version for your operating system from: [GitHub](https://github.com/keptn/keptn/releases/tag/0.7.3)
+- Download the version for your operating system from: [GitHub](https://github.com/keptn/keptn/releases/tag/0.8.0)
 - Unpack the archive
 - Find the `keptn` binary in the unpacked directory
 
   - *Linux / macOS*: Add executable permissions (``chmod +x keptn``), and move it to the desired destination (e.g. `mv keptn /usr/local/bin/keptn`)
 
   - *Windows*: Copy the executable to the desired folder and add the executable to your PATH environment variable.
 
-- Now, you should be able to run the Keptn CLI: 
+- Now, verify that the installation has worked and that the version is correct by running:
     - *Linux / macOS*
 
     ```console
-    keptn --help
+    keptn version
     ```
     
     - *Windows*
 
     ```console
-    .\keptn.exe --help
+    keptn version
```

Comment:
the `.\` in front was here on purpose, as often on Windows we not rely on the user to install the CLI correctly.

Reply:
I agree, without setting the PATH variable you have to reference the exe. I just tested it on my local machine and it works with `.\` 
I will revert my changes accordingly.